packages/members-importer/lib/importer.js

```diff
@@ -23,8 +23,9 @@ module.exports = class MembersCSVImporter {
      * @param {({name, at, job, data, offloaded}) => void} options.addJob - Method registering an async job
      * @param {Object} options.knex - An instance of the Ghost Database connection
      * @param {Function} options.urlFor - function generating urls
+     * @param {Object} options.context
      */
-    constructor({storagePath, getTimezone, getMembersApi, sendEmail, isSet, addJob, knex, urlFor}) {
```

Comment:
Do we have context at the time the importer is created? I think this would need to be passed into the function performing the import?

Reply:
@ErisDS Yes, I do have changes upcoming for the Ghost side of it as well. My commit for this is [here](https://github.com/aileen/Ghost/commit/9ece9d1025d7c49e25e62dcb9923216f95a1fac8) and I got inspired using it in the constructor by seeing how we handle context e. g. for the data importer [here](https://github.com/aileen/Ghost/blob/a5484a0fe12cc7ec8e7575deb240b598cb8c567f/core/server/data/importer/importers/data/data-importer.js#L44-L46). 

I honestly am not sure about doing this. We could also just hardcode it within the importer, but seeing the usage of context in the model layer, I went for this option.

Let me know what your thoughts are on this, I'm really not 100% sure.

Reply:
Sorry yes you are correct this is fine.